src/vivarium/examples/disease_model/population.py

```diff
@@ -144,5 +143,5 @@ def on_time_step(self, event: Event) -> None:
             information.
         """
         population = self.population_view.get_private_columns(event.index, "age", query="alive == 'alive'")
-        population += event.step_size / pd.Timedelta(days=365)
+        population += event.step_size / pd.Timedelta(days=365)  # type: ignore[operator]
```

Comment:
would coercing both sides to float work?

Reply:
We've just been ignoring issues with the step size and clock, since it can be either an int or a time object and it's a mess.